src/include/security.h

```diff
@@ -138,6 +138,19 @@ pgagroal_user_known(char* user);
 int
 pgagroal_tls_valid(void);
 
+/**
+ * @brief Generate a random ASCII password have size of pwd_length
+ * @param pwd_length length of the password
+ * @return Generated password
+ */
+char*
+generate_password(int pwd_length);
```

Comment:
Functions in `.h` files need to be prefixed with `pgagroal_`.

For this case - I think it would be better to do

```
int
pgagroal_generate_password(int password_length, char** password);
```